fineract-provider/src/main/java/org/apache/fineract/accounting/glaccount/data/GLAccountData.java

```diff
@@ -252,22 +247,6 @@ public GLAccountData(final Long id, final String name, final String glCode) {
         this.allowedExpensesTagOptions = null;
     }
 
-    public Long getId() {
-        return this.id;
-    }
-
-    public String getName() {
-        return this.name;
-    }
-
-    public String getGlCode() {
-        return this.glCode;
-    }
-
-    public EnumOptionData getType() {
-        return this.type;
-    }
-
     public Integer getTypeId() {
```

Comment:
I know this is not your code @hmg65 ... but: this is non-sense code... "a boxed basic type value that get's nullchecked to be unboxed just to return it as a boxed value"...  the nicest thing I can say about a (anti-)pattern like this is "how convoluted" (and I know we have some of them pretty much everywhere... not the last you'll see). So, when we see stuff like this let's simplify... remove it.